client/post-editor/editor-title/index.jsx

```diff
@@ -52,40 +56,39 @@ export default React.createClass( {
 			const input = ReactDom.findDOMNode( this.refs.titleInput );
 			input.focus();
 		}
-	},
+	}
 
-	onChange( event ) {
+	onChange = event => {
 		if ( ! this.props.post ) {
 			return;
 		}
 
-		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
-		PostActions.edit( {
+		this.props.editPost( this.props.site.ID, this.props.post.ID, {
```

Comment:
Example of why passing `siteId` is often preferable when possible:

```
index.jsx:212 Uncaught (in promise) TypeError: Cannot read property 'ID' of null(…)
```

When loading editor after `localStorage.clear()` + refresh.